Database migrations

Daste745 opened the pull request:
Use `prisma migrate` to store and manage database migrations.

Unfortunately, we have to run some goofy commands in order to load the env file and use the correct package (`/packages/db/`). We should probably fix this in the future to make stuff cleaner, but for now it should suffice.

Create a migration:
`bunx prisma migrate dev --schema packages/db/prisma/schema.prisma`

Apply migrations:
`bunx prisma migrate deploy --schema packages/db/prisma/schema.prisma`

Closes #16

Daste745 committed:
Create an initial database migration

Daste745 committed:
Update the README to include project setup tests

HKGx reviewed (approved):
I believe we could move `bunx prisma migrate dev --schema packages/db/prisma/schema.prisma` to a package script. 🤔
If you're willing to spend some time - do it, otherwise LGTM 🚀

Daste745 committed:
Add database migration scripts to the root package.json

This uses the system `bun` installation, instead of our `packages/db`'s
prisma. Unfortunately, package scripts in workspaces are very goofy, so
this might be the only viable option for now.
